src/components/admin/applications/AllApplicationsTable.tsx

```diff
@@ -46,6 +46,10 @@ const AllApplicationsTable: React.FC = () => {
   const [applicationBranch, setApplicationBranch] = useState<string[]>([]);
   const [confirmationBranch, setConfirmationBranch] = useState<string[]>([]);
 
+  const [width, setWidth] = useState<number>(window.innerWidth);
+
+  const isMobile = width <= 768;
```

Comment:
So this is not gonna work for the width as the state won't update as the screen size changes. What you can do is keep it a normal stack, and then on the Stack component, add this `flexDirection={{ base: "row", md: "column" }}`